src/App.tsx

```diff
@@ -1,17 +1,35 @@
-import { PokemonsList } from './PokemonsList';
+import { Game } from './Game';
 import './App.css';
 import { useState } from 'react';
+import { Route, BrowserRouter, Switch } from 'react-router-dom';
+import { MainPage } from './MainPage';
 
 export function App(): JSX.Element {
-  const [selectedPokemons, setSelectedPokemons] = useState<string[]>([]);
+  const [selectedPokemons, setSelectedPokemons] = useState<[string, string]>([
+    '',
+    '',
+  ]);
 
   return (
     <div className="App">
-      <h1 className="header-text">Choose two pokemons</h1>
-      <PokemonsList
-        selectedPokemons={selectedPokemons}
-        onChange={setSelectedPokemons}
-      />
+      <BrowserRouter>
+        <Switch>
+          <Route
+            exact
```

Comment:
move this route to the bottom, and remove `exact`

Check how react-router selects correct route